kits/agp1/contracts/AGP1Kit.sol

```diff
@@ -0,0 +1,86 @@
+pragma solidity 0.4.24;
+
+import "@aragon/apps-finance/contracts/Finance.sol";
+import "@aragon/apps-vault/contracts/Vault.sol";
+import "@aragon/apps-voting/contracts/Voting.sol";
+import "@aragon/apps-shared-minime/contracts/MiniMeToken.sol";
+
+import "@aragon/os/contracts/apm/APMNamehash.sol";
+import "@aragon/os/contracts/common/IsContract.sol";
+
+import "@aragon/kits-base/contracts/KitBase.sol";
+
+
+contract AGP1Kit is KitBase, APMNamehash, IsContract {
+
+    uint64 constant public MAIN_VOTING_SUPPORT = 50 * 10**16; // > 50%
+    uint64 constant public MAIN_VOTING_QUORUM = 0; // Just 1 vote is enough
+    uint64 constant public MAIN_VOTING_VOTE_TIME = 48 hours;
+
+    uint64 constant public META_TRACK_VOTING_SUPPORT = 666666666666666666; // > two thirds
+    uint64 constant public META_TRACK_VOTING_QUORUM = 0; // Just 1 vote is enough
+    uint64 constant public META_TRACK_VOTING_VOTE_TIME = 48 hours;
+
+    uint64 constant public FINANCE_PERIOD_DURATION = 7889400; // 365.25 days / 4
+
+    bytes32 constant public financeAppId = apmNamehash("finance");
+    bytes32 constant public vaultAppId = apmNamehash("vault");
+    bytes32 constant public votingAppId = apmNamehash("voting");
```

Comment:
We don't really _need_ any of these constants to be public, but it probably doesn't matter here.

Reply:
For kits i like appIds to be public as it helps sanity check and debugging